doc/nrf/ug_matter_device_certification.rst

```diff
@@ -7,12 +7,225 @@ Matter certification
+
+Joining CSA allows you to meet the following requirements:
+
+* Familiarity with the Matter specification, which is the standard for the certification.
+  You can access the specification from the `Matter Resource Kit`_ after you join CSA.
+* Assignment of Vendor ID (VID), which is required when applying for certification.
+  You can apply to `CSA's Certification Team`_ to obtain VID.
+  The VID codes are valid immediately upon assignment.
+
+.. _ug_matter_device_certification_testing:
+
+Testing
+=======
+
+The Matter certification process requires testing before any application for certification is sent.
+The testing can either be performed at an Authorized Test Laboratory (ATL), using a test plan developed by CSA, or at a Specification Validation Event (SVE).
+
+Testing at an ATL
+-----------------
+
+Each ATL has its own submission process, but all require a Declaration of Conformity and a Protocol Implementation Conformance Statement (PICS).
+Both documents confirm that the Matter component you are going to certify is meeting the protocol requirements.
+
+The ATL can contact you if your Matter component is failing any of the test cases.
+
+You can find the list of available ATLs on the `CSA's Testing Providers`_ page.
+
+Testing at an SVE
+-----------------
+
+You can complete the required testing at a Specification Validation Event.
+Such events are regularly organized by CSA to verify conformity with the Matter specification.
+
+If your Matter component passes all of the testing phases, including the final Gating Test Event, no additional testing at an ATL is required.
+However, an ATL still has to validate the SVE testing results during the `Reporting`_ step.
+
+.. _ug_matter_device_certification_reporting:
+
+Reporting
+=========
+
+After the Matter component successfully passes all required test cases, the ATL sends a test report to both you and CSA.
+The test report summarizes the testing process and includes a variety of information, including details about the device, `Certification types`_, test plans, and Test Harness.
+
+Application
+===========
+
+Once you have obtained the test report for your Matter component, you can apply for the certification to CSA.
+You can do this online from the `CSA Member Resources page`_ using the different Certification and Testing tools to submit the required documentation.
+
+Application requirements
+------------------------
+
+The final application for certification requires you to send the following documents to CSA:
+
+* Declaration of Conformity
+* `PICS`_
+* `Dependent Transport Attestation`_
+* `Matter Attestation of Security`_
+* Description of the Matter component to be certified (and photos of the end product, for the product certification)
+
+PICS
+++++
+
+You can generate the PICS in the XML format using CSA's PICS Tool, available from `CSA Member Resources page`_.
```

Comment:
@doublemis1 Is PICS an acronym? Could we at least explain what it stands for?

Reply:
It is explained earlier in the text.